core/src/test/scalajvm/sttp/tapir/generic/MultipartCodecDerivationTestExtensionsJVM.scala

```diff
@@ -0,0 +1,17 @@
+package sttp.tapir.generic
+
+import java.io.File
+
+trait MultipartCodecDerivationTestExtensionsJVM { self: MultipartCodecDerivationTestJVM =>
```

Comment:
hmm can't this go into the main test? Why a separate `-JVM` trait? these are all